src/llmcompressor/modifiers/obcq/utils/sgpt_wrapper.py

```diff
@@ -198,6 +209,8 @@ def compress(
         if isinstance(self.layer, transformers.Conv1D):
             W = W.t()
         W = W.reshape(final_shape).to(final_dtype)
+        if weight_quant_args is not None:
+            W = maybe_calibrate_or_quantize(self.layer, W, "weight", weight_quant_args)
```

Comment:
why do we need this additional call to maybe_calibrate_or_quantize?

Reply:
the sparseGPT algorithm adjusts the weights and they aren't guaranteed to be in the quantized bins, we need to requantize after updating